backend/src/room/room.service.ts

```diff
@@ -289,9 +304,18 @@ export class RoomService
     if (members.length === 0) {
       return null;
     }
-    return members.map((info) => ({
-      userId: Number(info.split(RoomService.DELIMITER)[0]),
-      isConnected: Boolean(Number(info.split(RoomService.DELIMITER)[1])),
+    const map = new Map<number, boolean>();
+    for (const member of members) {
+      const userId = Number(member.split(RoomService.DELIMITER)[0]);
+      const isConnected =
+        map.get(userId) ||
+        false ||
```

Comment:
```suggestion
```

Reply:
Need to convert the undefined to false 🥲

Reply:
Oh oops I thought it wasn't needed cuz there is a boolean expression behind 😮

Reply:
Oh right that part totally flew over my head.